guardrails/guard.py

```diff
@@ -1219,7 +1219,7 @@ def _save(self):
         if api_key is not None:
             if self.name is None:
                 self.name = f"gr-{str(self.id)}"
-                logger.warn("Warning: No name passed to guard!")
+                logger.warn("No name passed to guard!")
```

Comment:
Let's switch this to `logger.warning`. Reason: I _think_ `logging.warn` is deprecated.

Reply:
TIL. thanks 👍🏼